pages/components/employee-dashboard.js

```diff
@@ -0,0 +1,44 @@
+// Creating children: Query from database, create children element
+// and append too appropriate column
+
+
+
+////////////////////////////////////
+// Selecting children
```

Comment:
Expecting AJAX to query data from db. This section needs to:
+Query raw data
+Create jquery variable for #task-id, #task-desc, #deadline, #assigners
+Append data